--- TiMidity++-2.13.2/configure	2014-01-31 15:28:45.456420449 +0000
+++ TiMidity++-2.13.2/configure	2014-01-31 15:30:05.719751398 +0000
@@ -15814,13 +15814,13 @@
 	;;
     esac
 
-    echo "$as_me:$LINENO: checking for initscr in -lncurses" >&5
-echo $ECHO_N "checking for initscr in -lncurses... $ECHO_C" >&6
+    echo "$as_me:$LINENO: checking for initscr in $(pkg-config --libs ncurses)" >&5
+echo $ECHO_N "checking for initscr in $(pkg-config --libs ncurses)... $ECHO_C" >&6
 if test "${ac_cv_lib_ncurses_initscr+set}" = set; then
   echo $ECHO_N "(cached) $ECHO_C" >&6
 else
   ac_check_lib_save_LIBS=$LIBS
-LIBS="-lncurses  $LIBS"
+LIBS="$(pkg-config --libs ncurses) $LIBS"
 cat >conftest.$ac_ext <<_ACEOF
 /* confdefs.h.  */
 _ACEOF
@@ -15879,7 +15879,7 @@
 echo "$as_me:$LINENO: result: $ac_cv_lib_ncurses_initscr" >&5
 echo "${ECHO_T}$ac_cv_lib_ncurses_initscr" >&6
 if test $ac_cv_lib_ncurses_initscr = yes; then
-  lib_curses_opt=-lncurses
+  lib_curses_opt="$(pkg-config --libs ncurses)"
 else
    	echo "$as_me:$LINENO: checking for PDC_set_ctrl_break in -lcurses" >&5
 echo $ECHO_N "checking for PDC_set_ctrl_break in -lcurses... $ECHO_C" >&6
--- TiMidity++-2.13.2/configure.in	2014-01-31 15:28:51.253086944 +0000
+++ TiMidity++-2.13.2/configure.in	2014-01-31 15:30:29.833084016 +0000
@@ -1446,7 +1446,7 @@
 	;;
     esac
 
-    AC_CHECK_LIB(ncurses,initscr,lib_curses_opt=-lncurses,
+    AC_CHECK_LIB(ncurses,initscr,lib_curses_opt="$(pkg-config --libs ncurses)",
       [ dnl checking pdcurses
 	AC_CHECK_LIB(curses,PDC_set_ctrl_break,
           [ lib_curses_opt=-lcurses
