--- bcl2fastq/src/perl/lib/Casava/Alignment/Config.pm	2014-04-16 14:33:45.000000000 +0100
+++ bcl2fastq/src/perl/lib/Casava/Alignment/Config.pm	2014-04-16 14:32:16.000000000 +0100
@@ -744,17 +744,17 @@
 {
     my $self = shift;
     my ($project, $sample, $lane, $barcode, $reference) = @_;
-    for my $variable qw(ELAND_FASTQ_FILES_PER_PROCESS)
+    for my $variable (qw(ELAND_FASTQ_FILES_PER_PROCESS))
     {
         $self->_validateExistence($variable, $project, $sample, $lane, $barcode, $reference);
     }
-    for my $directory qw(ELAND_GENOME)
+    for my $directory (qw(ELAND_GENOME))
     {
         $self->_validateDirectory($directory, $project, $sample, $lane, $barcode, $reference);
     }
 }
 
-sub _isBooleanTrue($)
+sub _isBooleanTrue
 {
     my $value = shift;
     return $value =~ /^y$|^yes$|^on$|^true$|^1$|^ok$/i;
