diff -Nuar util-linux-2.24.orig/configure util-linux-2.24/configure
--- util-linux-2.24.orig/configure	2014-01-31 18:18:14.366118443 +0000
+++ util-linux-2.24/configure	2014-01-31 18:20:31.549447702 +0000
@@ -18040,13 +18040,13 @@
 
 
   have_ncurses=yes
-  { $as_echo "$as_me:${as_lineno-$LINENO}: checking for initscr in -lncursesw" >&5
-$as_echo_n "checking for initscr in -lncursesw... " >&6; }
+  { $as_echo "$as_me:${as_lineno-$LINENO}: checking for initscr in $(pkg-config --libs ncursesw)" >&5
+$as_echo_n "checking for initscr in $(pkg-config --libs ncursesw)... " >&6; }
 if ${ac_cv_lib_ncursesw_initscr+:} false; then :
   $as_echo_n "(cached) " >&6
 else
   ac_check_lib_save_LIBS=$LIBS
-LIBS="-lncursesw  $LIBS"
+LIBS="$(pkg-config --libs ncursesw) $LIBS"
 cat confdefs.h - <<_ACEOF >conftest.$ac_ext
 /* end confdefs.h.  */
 
@@ -18081,7 +18081,7 @@
 #define HAVE_LIBNCURSESW 1
 _ACEOF
 
-  LIBS="-lncursesw $LIBS"
+  LIBS="$(pkg-config --libs ncursesw) $LIBS"
 
 else
   have_ncurses=no
@@ -18110,7 +18110,7 @@
 
 done
 
-	  NCURSES_LIBS="-lncursesw"
+	  NCURSES_LIBS="$(pkg-config --libs ncursesw)"
 
 fi
 
@@ -18120,13 +18120,13 @@
 
 
   have_ncurses=yes
-  { $as_echo "$as_me:${as_lineno-$LINENO}: checking for initscr in -lncurses" >&5
-$as_echo_n "checking for initscr in -lncurses... " >&6; }
+  { $as_echo "$as_me:${as_lineno-$LINENO}: checking for initscr in $(pkg-config --libs ncurses)" >&5
+$as_echo_n "checking for initscr in $(pkg-config --libs ncurses)... " >&6; }
 if ${ac_cv_lib_ncurses_initscr+:} false; then :
   $as_echo_n "(cached) " >&6
 else
   ac_check_lib_save_LIBS=$LIBS
-LIBS="-lncurses  $LIBS"
+LIBS="$(pkg-config --libs ncurses) $LIBS"
 cat confdefs.h - <<_ACEOF >conftest.$ac_ext
 /* end confdefs.h.  */
 
@@ -18161,7 +18161,7 @@
 #define HAVE_LIBNCURSES 1
 _ACEOF
 
-  LIBS="-lncurses $LIBS"
+  LIBS="$(pkg-config --libs ncurses) $LIBS"
 
 else
   have_ncurses=no
@@ -18178,7 +18178,7 @@
 
 	if test "x$have_ncurses" = xyes; then :
 
-	  NCURSES_LIBS="-lncurses"
+	  NCURSES_LIBS="$(pkg-config --libs ncurses)"
 
 fi
 
diff -Nuar util-linux-2.24.orig/configure.ac util-linux-2.24/configure.ac
--- util-linux-2.24.orig/configure.ac	2014-01-31 18:18:14.406118441 +0000
+++ util-linux-2.24/configure.ac	2014-01-31 18:20:54.446113689 +0000
@@ -633,13 +633,13 @@
 	UL_CHECK_LIB([ncursesw], [initscr], [ncurses])
 	AS_IF([test "x$have_ncurses" = xyes], [
 	  AC_CHECK_HEADERS([ncursesw/ncurses.h])
-	  NCURSES_LIBS="-lncursesw"
+	  NCURSES_LIBS="$(pkg-config -libs ncursesw)"
 	])
       ])
       AS_IF([test "x$have_ncurses" = xno], [
 	UL_CHECK_LIB(ncurses, initscr)
 	AS_IF([test "x$have_ncurses" = xyes], [
-	  NCURSES_LIBS="-lncurses"
+	  NCURSES_LIBS="$(pkg-config --libs ncurses)"
 	])
       ])
     ])
